### PR TITLE
ux update

### PR DESCRIPTION
**Pull request by mcgill-a**
Implemented the UX changes from Ryan's initial proposal.

desktop
mobile
end

### CODE REVIEW

**Review by athompsonScottLogic — APPROVED** [review]
looks good to go when conflicts are resolved

**Comment by mcgill-a** [comment]
Merge conflicts are now resolved + auto formatted for consistent tab spacing + layout

**Comment by mcgill-a** [comment]
Merge conflicts are now resolved + auto formatted for consistent tab spacing + layout

**Comment by mcgill-a** [comment]
image

**Review by athompsonScottLogic — APPROVED** [review]
good job looks good to go

**Comment by PHardySL** [comment]
looks good, can we speed up the transition from cards to result box?
assume a way to switch mode will be re-implemented?

File: whichone/static/js/ui.js [hunk 1]
```diff
@@ -1,170 +1,172 @@
...
-  countdown = 10;
-  countdownNumberEl.textContent = countdown;
-  if (refreshIntervalId != null) {
-    clearInterval(refreshIntervalId);
-  }
-  refreshIntervalId = setInterval(cdwn, 1000);
-  $("#inner-circle").addClass("animated");
+    $("#countdown-number").css("display", "block");
+    $("#inner-circle").css("display", "block");
+    countdown = 12;
```
Comment (connor-settle) [inline whichone/static/js/ui.js]: This should probably be a injectable config so you only need to change it in 1 place in future if you change the countdown timer again.

File: whichone/templates/feedback.html [hunk 2]
```diff
@@ -16,53 +15,44 @@
     {% block styles -%}
     <link rel="stylesheet" type="text/css"
         href="https://cdn.rawgit.com/mfd/f3d96ec7f0e8f034cc22ea73b3797b59/raw/856f1dbb8d807aabceb80b6d4f94b464df461b3e/gotham.css">
-    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/main.css')}}">
-    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/skeleton.css')}}">
-    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/normalize.css')}}">
+    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/redesign.css')}}">
```
Comment (connor-settle) [inline whichone/templates/feedback.html]: What does redesign represent? Is it still the main.css?

**Review by connor-settle — APPROVED** [review]
Looks good

**Review by PHardySL — APPROVED** [review]
wow, looks a lot cleaner now!

File: whichone/templates/feedback.html (hunk 2 above)
Comment (mcgill-a) [reply whichone/templates/feedback.html]: It's a re-written version of main.css that has way more reusable components. It still contains most of the css so we could rename it back to main

**Pull request merged**